### ISSUE TITLE
Support for "type arguments ... conform to the bounds of none ..."

### ISSUE THREAD

**Issue by kciesielski** [softwaremill/scala-clippy#9]
Example from Slick 3.1.1:
```
XXXX.scala:24: type arguments [com.company.SomeType] conform to the bounds of none of the overloaded alternatives of
[error]  value apply: [E <: slick.lifted.AbstractTable[_]]=> slick.lifted.TableQuery[E] <and> [E <: slick.lifted.AbstractTable[_]](cons: slick.lifted.Tag => E)slick.lifted.TableQuery[E]
[error]   protected val value = TableQuery[SomeType]
[error]                                ^
[error] one error found
```
Advice should be something like:
```
Verify whether you define your table as "class SomeType(tag: Tag) extends Table[SomeType]", 
specifically please double check whether you passed correct type to Table[]. 
A typical mistake is specifying two classes, like case class User and class 
Users to represent DB table, and then trying to create a Table[User]
 instead of Table[Users]
```

**Comment by adamw** [comment]
Should be available in 0.2.4. Can you try submitting advice?

**Comment by kciesielski** [comment]
Looks like it works, thanks :)
